convert_state.py

```diff
@@ -1,60 +1,59 @@
+"""Program that converts state names to abbreviations and vice versa"""
 
+import json
 
 
-import json  # import json for json processing
 def main():
-    with open('data.json','r') as file:
-        data = file.read()
-        data = json.loads(data)
+    """Reading in json data"""
 
-    print(data)
+    with open('data.json', 'r') as file:
+        state_json = file.read()
+        abbr_to_names = json.loads(state_json)
```

Comment:
Avoid abbreviations in variables names, use abbreviation_to_names. If I know the name is abbreviated, is it abbr_to_names or abrv_to_names or ? Autocomplete helps with longer names so it isn't more typing.